android/cli/commands/_build.js

```diff
@@ -2510,6 +2510,113 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 		}
 	}, this);
 
+	// genereate missing app icons
+	tasks.push(function (cb) {
+		this.currentBuildManifestFiles = this.currentBuildManifestFiles || {};
```

Comment:
Android build doesn't use differential builds... yet. There is no need for `this.currentBuildManifestFiles`. All references should be removed.